include/alpaka/core/Common.hpp

```diff
@@ -186,7 +192,7 @@
     || BOOST_LANG_HIP)
 #    define ALPAKA_STATIC_ACC_MEM_CONSTANT                                                                            \
         template<typename TAcc>                                                                                       \
-        inline __constant__
+        __constant__
```

Comment:
Can you do the same as `__device__` also for `__constant__` ?

Reply:
done!